lua/swamp/misc/sh_titles.lua

```diff
@@ -160,6 +160,69 @@ AddTitle("kleinertp", {
     {1, "Test Subject", 10000}
 }, "Be subjected to one of Dr. Isaac Kleiner's teleportation experiments", "s_kleinertp")
 
+AddTitle("buttonsearch", {
+    { 1, "Button Rookie", 0},
+    { 10, "Button Hunter", 0},
+    { 100, "Button Master", 0}
+}, "Find and press %s buttons.", "s_magicbutton")
+
+AddTitle("snowballhit", {
```

Comment:
probably a bad idea since snowballs are only available during december

Reply:
> probably a bad idea since snowballs are only available during december

So is giving gifts though?

Reply:
No, the mystery box is a gift. Probably can make the snowball appear in the mysterybox to address this...

Reply:
Aren't you getting snowballs when you stand on the vents on the roofs?